packages/package-requester/src/packageRequester.ts

```diff
@@ -476,10 +479,11 @@ function fetchToStore (
             }
           }),
       )
-      await writeJsonFile(path.join(target, 'integrity.json'), integrity, { indent: undefined })
+      await writeJsonFile(pkgIndexFilePath, integrity, { indent: undefined })
```

Comment:
Passing `indent: undefined` shouldn't be necessary if `writeJsonFile` uses nullish coalescing to check that value.